### ISSUE TITLE
moveCursorTo isn't working as expected

### ISSUE THREAD

**Issue by reubenfirmin** [ajaxorg/ace#5412]
### Describe the bug

The behavior of moveCursorTo has apparently changed. https://ace.c9.io/api/classes/Ace.Editor.html#moveCursorTo

### Expected Behavior

moveCursorTo(3, 0) should move to line 3. 

### Current Behavior

It instead moves to line 0.

### Reproduction Steps

Here is a jsFiddle (not mine) which apparently used to work, and no longer does. https://jsfiddle.net/q27ebeeq/130/

No matter how you interact with this demo, the cursor is always reset to 0,0.

### Ace Version / Browser / OS / Keyboard layout

The fiddle is on 1.3.1. I'm on 1.30.0. Both show the same issue.

**Comment by nightwing** [comment]
The behavior of moveCursorTo have not changed, there is an old bug in ui-ace not updating field.value and the line `$scope.field.val = $scope.editor.getValue();` moving ace cursor back to session start.

You can add the following code to see this
```js
    $scope.editor.session.setValue = function(v) {
    	if (this.getValue()!=v)
    		this.__proto__.setValue(v)
    }
```

Btw is the ui-ace package maintained, the repository is archived and issues have not been replied to in a while?

**Comment by andredcoliveira** [comment]
@reubenfirmin could you please confirm whether the issue is on Ace's side?

**Comment by reubenfirmin** [comment]
This actually seems to be some change in behaviour w.r.t. keyhandlers. I was doing everything in a keyhandler, and returning true or false instead of a command. (The docs...are not good. https://ajaxorg.github.io/ace-api-docs/classes/Ace.KeyboardHandler.html. I think I picked up incorrect info from other stackoverflow posts in the past).

I moved various key combos to commands, and now things work as expected again.
